client/events.lua

```diff
@@ -220,17 +220,4 @@ RegisterNetEvent('QBCore:Command:ShowMe3D', function(senderId, msg)
             Wait(0)
         end
     end)
-end)
-
--- Listen to Shared being updated
-RegisterNetEvent('QBCore:Client:OnSharedUpdate', function(tableName, key, value)
-    QBCore.Shared[tableName][key] = value
-    TriggerEvent('QBCore:Client:UpdateObject')
-end)
-
-RegisterNetEvent('QBCore:Client:OnSharedUpdateMultiple', function(tableName, values)
-    for key, value in pairs(values) do
-        QBCore.Shared[tableName][key] = value
-    end
-    TriggerEvent('QBCore:Client:UpdateObject')
-end)
```

Comment:
Is this a breaking change, or are these "internal" events? If breaking, add a "!" to PR name. https://www.conventionalcommits.org/en/v1.0.0/

Reply:
These are "internal", besides the update object, but it's not really breaking because you won't notice it at all in other scripts and it doesn't break anything by having it removed